src/components/DataTable/data/DataTable.tsx

```diff
@@ -0,0 +1,31 @@
+
+// The data to populate the table in Storybook
+
+import type {UserDataProps} from '../types/UserData.types';
```

Comment:
Since we have a simple dataset, TypeScript can infer the typing, so you don't have to specify it here. You can get rid of `UserData.types`